cli/dcoscli/package/main.py

```diff
@@ -377,10 +377,7 @@ def _install(package_name, package_version, options_path, app_id, cli, app,
 
         emitter.publish(msg)
 
-        package_manager.install_app(
-            pkg,
-            options,
-            app_id)
+        package_manager.install_app(pkg, user_options, app_id)
```

Comment:
So, `options` previously was returned from `pkg.options(user_options)`. If you look at the definition of that method, it just returns its argument unchanged (with the side effect of sending it in a request to Cosmos).

Maybe I should update that method to not return anything. What do you think?